packages/flutter/test/widgets/reorderable_list_test.dart

```diff
@@ -0,0 +1,198 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+void main() {
+  testWidgets('ReorderableList, drag and drop, fixed height items', (WidgetTester tester) async {
```

Comment:
Not sure, but I think it's:
```suggestion
  testWidgets('SliverReorderableList, drag and drop, fixed height items', (WidgetTester tester) async {
```